### ISSUE TITLE
is this still being worked on?

### ISSUE THREAD

**Issue by Orifan1** [Larcondos/Custom-Robo-Unity#5]
im curious, and there's nothing in issues, pull requests, actions, or projects so i cant exactly date the last time this was touched.
sorry if this is annoying

**Comment by xlilos** [comment]
Yeah I had the same thought, seems like this repo/profile has been left for awhile. Seems like this started as a college project that probably got left alone after the class ended. Fair enough, I have done the same.

Anyway I am going to poke around this code for a bit. I started messing with Unity to maybe do something similar to this as a passion project. I haven't done too much game dev but have always wanted to learn more. Different dev vibes compared to my normal software dev.
